docs/product/components/popovers.html

```diff
@@ -281,6 +287,97 @@
+    <div class="stacks-preview">
+{% highlight javascript %}
+Stacks.application.register("section", class extends Stacks.StacksController {
+    static targets = ["help"];
+
+    showHelp(event) {
+         Stacks.showPopover(this.helpTarget);
+         event.stopPropagation();
+    }
+    hideHelp(event) {
+         Stacks.hidePopover(this.helpTarget);
+    }
+});
+{% endhighlight %}
+        <div class="stacks-preview--example bg-black-025">
+            <div data-controller="section">
+                <div class="s-page-title mb16">
+                    <div class="s-page-title--text">
+                        <h1 class="s-page-title--header">Lorem ipsum</h1>
+                    </div>
+                    <div class="s-page-title--actions">
+                        <button type="button" class="s-btn s-btn__muted p0"
+                                data-target="section.help"
+                                data-controller="s-popover"
+                                data-action="s-popover#toggle"
+                                data-s-popover-placement="left"
+                                aria-controls="popover-help-example">{% icon "Help" %}</button>
+                        <div class="s-popover wmn-initial w-auto"
+                             id="popover-help-example"
+                             role="menu">
+                             <button class="s-popover--close s-btn s-btn__muted" data-action="section#hideHelp" aria-label="Close">{% icon "ClearSm" %}</button>
+                             <div class="s-popover--arrow"></div>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                        </div>
+                    </div>
+                </div>
+                <p class="fc-light">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et metus molestie nulla luctus sodales ac luctus justo. Aenean iaculis ac ante sit amet aliquam. Duis dolor velit, imperdiet sed mauris eu, sollicitudin egestas nisl. Ut vitae nulla eu risus iaculis semper sit amet vitae tortor. Sed convallis lacus quis libero placerat finibus. Phasellus pulvinar vel nunc eu tempor. Sed pharetra magna a felis egestas placerat. Sed imperdiet dui a sem fermentum, eget consectetur elit feugiat. Phasellus non condimentum orci. Nam id molestie elit, ut gravida metus. Vivamus vel nunc risus. Maecenas posuere sit amet tellus vel laoreet. Nulla lacus mauris, rhoncus eu laoreet quis, mattis vehicula nisl. Integer efficitur quam et nisi luctus scelerisque. Mauris efficitur lectus ac malesuada congue.</p>
+                <button class="s-btn s-btn__link" data-action="section#showHelp">Click here for help</button>
+            </div>
+        </div>
+    </div>
+    <script>
+        window.addEventListener("load", () => {
```

Comment:
Ugh... This is only firing if you land on the page directly, not navigating to it.  I forget what the right approach is here.

Reply:
Ah, right right right. Yep. Toss this into a JS file and load it with `defer`. It's really great as a consumer of the docs to load everything via AJAX but pretty much universally trolls every new feature demo we build, lol.

Reply:
Added to `feature.popovers.js`